Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -8,22 +8,22 @@
 
 namespace Xamarin.Forms
 {
-	public abstract class Cell : Element, ICellController, IElementConfiguration<Cell>, IFlowDirectionController, IPropertyPropagationController, IVisualController
+	// Don't add IElementConfiguration<Cell> because it kills performance on UWP structures that use Cells
```

Comment:
this is not a code comment, because as soon as the patch is applied it will be out of date and refer to something that's no longer there :)

but that's a good PR comment to add, for history purposes

Reply:
@StephaneDelcroix  I don't quite follow. The main purpose of the comment is so that future generations won't add the interface to Cell and run into this same issue.  Would it make more sense if I left a commented out version of IElementConfiguration<cell> so that there is more context?